middlewares/articleValidation.js

```diff
@@ -22,6 +22,21 @@ export default {
     return next();
   },
 
+  categoryQueryValidator(req, res, next) {
+    const { category } = req.query;
+    if (category === undefined) {
+      return next();
```

Comment:
This is not tested. Write a test for when category is undefined.

Reply:
Ok. Doing it right away